<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="buli" ignored="1" name="all" tests="36" failures="4" timestamp="25 九月 2020 03:14:04 GMT" time="4.176" errors="0">
  <testcase name="exit" time="0.001" classname="com.trustkernel.kpp.api.testcase.AnnotationsTest"/>
  <testcase name="login" time="0.0" classname="com.trustkernel.kpp.api.testcase.AnnotationsTest"/>
  <testcase name="getUser" time="0.0" classname="com.trustkernel.kpp.api.testcase.DataProviderTest"/>
  <testcase name="getUser" time="0.0" classname="com.trustkernel.kpp.api.testcase.DataProviderTest"/>
  <testcase name="getUser" time="0.0" classname="com.trustkernel.kpp.api.testcase.DataProviderTest"/>
  <testcase name="getUsers" time="0.078" classname="com.trustkernel.kpp.api.testcase.DataProviderTest"/>
  <testcase name="manyDataProviderTest" time="0.002" classname="com.trustkernel.kpp.api.testcase.DataProviderTest"/>
  <testcase name="manyDataProviderTest" time="0.001" classname="com.trustkernel.kpp.api.testcase.DataProviderTest"/>
  <testcase name="manyDataProviderTest" time="0.001" classname="com.trustkernel.kpp.api.testcase.DataProviderTest"/>
  <testcase name="manyDataProviderTest" time="0.001" classname="com.trustkernel.kpp.api.testcase.DataProviderTest"/>
  <testcase name="oneDataProviderTest" time="0.0" classname="com.trustkernel.kpp.api.testcase.DataProviderTest"/>
  <testcase name="oneDataProviderTest" time="0.0" classname="com.trustkernel.kpp.api.testcase.DataProviderTest"/>
  <testcase name="oneDataProviderTest" time="0.001" classname="com.trustkernel.kpp.api.testcase.DataProviderTest"/>
  <testcase name="oneDataProviderTest" time="0.001" classname="com.trustkernel.kpp.api.testcase.DataProviderTest"/>
  <testcase name="twoDataProviderTest" time="0.001" classname="com.trustkernel.kpp.api.testcase.DataProviderTest"/>
  <testcase name="twoDataProviderTest" time="0.001" classname="com.trustkernel.kpp.api.testcase.DataProviderTest"/>
  <testcase name="twoDataProviderTest" time="0.001" classname="com.trustkernel.kpp.api.testcase.DataProviderTest"/>
  <testcase name="twoDataProviderTest" time="0.0" classname="com.trustkernel.kpp.api.testcase.DataProviderTest"/>
  <testcase name="dependGroupsTest1" time="0.0" classname="com.trustkernel.kpp.api.testcase.DependsOnMethodTest"/>
  <testcase name="dependGroupsTest2" time="0.0" classname="com.trustkernel.kpp.api.testcase.DependsOnMethodTest">
    <failure type="java.lang.AssertionError" message="error...">
      <![CDATA[java.lang.AssertionError: error...
at com.trustkernel.kpp.api.testcase.DependsOnMethodTest.dependGroupsTest2(DependsOnMethodTest.java:31)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- dependGroupsTest2 -->
  <testcase name="dependTest" time="0.001" classname="com.trustkernel.kpp.api.testcase.DependsOnMethodTest"/>
  <testcase name="arithmeticException" time="0.001" classname="com.trustkernel.kpp.api.testcase.ExceptionTest"/>
  <testcase name="runTimeExceptionFailed" time="0.0" classname="com.trustkernel.kpp.api.testcase.ExceptionTest">
    <failure type="org.testng.TestException" message="
Method ExceptionTest.runTimeExceptionFailed()[pri:0, instance:com.trustkernel.kpp.api.testcase.ExceptionTest@184f6be2] should have thrown an exception of type class java.lang.RuntimeException">
      <![CDATA[org.testng.TestException: 
Method ExceptionTest.runTimeExceptionFailed()[pri:0, instance:com.trustkernel.kpp.api.testcase.ExceptionTest@184f6be2] should have thrown an exception of type class java.lang.RuntimeException
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- runTimeExceptionFailed -->
  <testcase name="runTimeExceptionSuccess" time="0.0" classname="com.trustkernel.kpp.api.testcase.ExceptionTest"/>
  <testcase name="ignore2" time="0.0" classname="com.trustkernel.kpp.api.testcase.IgnoreTest"/>
  <testcase name="ignore3" time="0.0" classname="com.trustkernel.kpp.api.testcase.IgnoreTest"/>
  <testcase name="test1" time="0.0" classname="com.trustkernel.kpp.api.testcase.MultiThreadOnXmlTest"/>
  <testcase name="test2" time="0.0" classname="com.trustkernel.kpp.api.testcase.MultiThreadOnXmlTest"/>
  <testcase name="test3" time="0.0" classname="com.trustkernel.kpp.api.testcase.MultiThreadOnXmlTest"/>
  <testcase name="paramTest" time="0.0" classname="com.trustkernel.kpp.api.testcase.ParameterTest">
    <failure type="org.testng.TestNGException" message="
Parameter &amp;apos;name&amp;apos; is required by @Test on method paramTest but has not been marked @Optional or defined
in E:\APITestProject\kpp_ApiAutoFrame\testsuites\testng.xml">
      <![CDATA[org.testng.TestNGException: 
Parameter 'name' is required by @Test on method paramTest but has not been marked @Optional or defined
in E:\APITestProject\kpp_ApiAutoFrame\testsuites\testng.xml
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- paramTest -->
  <testcase name="testFailed" time="2.033" classname="com.trustkernel.kpp.api.testcase.TimeOutTest">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method com.trustkernel.kpp.api.testcase.TimeOutTest.testFailed() didn&amp;apos;t finish within the time-out 2000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method com.trustkernel.kpp.api.testcase.TimeOutTest.testFailed() didn't finish within the time-out 2000
]]>
    </failure>
  </testcase> <!-- testFailed -->
  <testcase name="testSuccess" time="2.001" classname="com.trustkernel.kpp.api.testcase.TimeOutTest"/>
  <testcase name="testAdd" time="0.001" classname="com.trustkernel.kpp.api.testcase.TsetNGSimpleTest"/>
  <testcase name="dependGroupsTest3" time="0.001" classname="com.trustkernel.kpp.api.testcase.DependsOnMethodTest"/>
  <testcase name="dependGroupsTest4" time="0.0" classname="com.trustkernel.kpp.api.testcase.DependsOnMethodTest">
    <skipped/>
  </testcase> <!-- dependGroupsTest4 -->
  <testcase name="dependTest2" time="0.001" classname="com.trustkernel.kpp.api.testcase.DependsOnMethodTest"/>
  <testcase name="ignore1" time="0.0" classname="com.trustkernel.kpp.api.testcase.IgnoreTest">
    <ignored/>
  </testcase> <!-- ignore1 -->
</testsuite> <!-- all -->
